d3d9ex/AutoFix.cpp

```diff
@@ -405,28 +321,29 @@ void MainContext::FF13_2_InitializeGameAddresses()
 	ff13_2_internal_res_h = ff13_2_internal_res_w + 1;
 }
 
-void MainContext::FF13_2_RemoveContinuousControllerScan() {
+void MainContext::FF13_2_RemoveContinuousControllerScan()
+{
 	if (!config.GetFFXIIIDisableIngameControllerHotSwapping()) {
 		PrintLog("Continuous controller scanning not disabled (config)");
 		return;
 	}
 	// Disable continuous controller scanning.
 
 	PrintLog("Removing game slow and synchronous controller continuous controller scanning...");
-	context.ChangeMemoryProtectionToReadWriteExecute(ff13_2_continuous_scan_instruction_address, 1);
 	// change a jne to jmp
-	*(uint8_t*)ff13_2_continuous_scan_instruction_address = 0xEB;
+	MemPatch::Fill(ff13_2_continuous_scan_instruction_address, 0xEB, 1);
 }
 
-void MainContext::FF13_2_AddHookIngameFrameRateLimitSetter() {
-	if (context.AreAlmostTheSame((float)context.config.GetFFXIIIIngameFrameRateLimit(), 0.0F)) {
-		PrintLog("Frame rate should not be changed (config = 0)");
+void MainContext::FF13_2_AddHookIngameFrameRateLimitSetter()
+{
+	if (config.GetFFXIIIIngameFrameRateLimit() == -1) {

```

Comment:
This should be ```if (config.GetFFXIIIIngameFrameRateLimit() == 0)``` (FF13-2 is broken when `IngameFrameRateLimit = 0` because of this).

`PrintLog` message below is also incorrect

Reply:
Yeah, missed that one